src/BlazorWebView/src/core/build/Microsoft.AspNetCore.Components.WebView.Maui.targets

```diff
@@ -0,0 +1,110 @@
+﻿<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!-- Enable Razor files to be visible in Solution Explorer because they are not Content files in .NET MAUI projects -->
+  <ItemGroup>
+    <AvailableItemName Include="RazorComponent"/>
```

Comment:
This is confusing to me - I'd expect to see something like `<AvailableItemName Include="@(RazorComponent)"/>`.

I guess my MSBuild skills are lacking here, because this must be making a useful difference or you wouldn't have added it.

Reply:
This is defining a list of strings that are VS will show in Solution Explorer (and also offer in the File Properties to change a component type). It's not adding the _files_ of that type to the list, rather just the string name itself.

Reply:
BTW here's the official example from the docs: https://docs.microsoft.com/en-us/visualstudio/msbuild/visual-studio-integration-msbuild?view=vs-2019#additional-build-actions (which doesn't mention this exact usage, but for which I filed a PR to update: https://github.com/MicrosoftDocs/visualstudio-docs/pull/6663)